### ISSUE TITLE
InvalidCommandError when defining a simple `paths_to_blueprints`

### ISSUE THREAD

**Issue by gonzalo-bulnes** [gonzalo-bulnes/dredd-rack#18]
**As a** developer 
**Given** I am testing a local API 
**And** I defined the blueprints location by providing a single argument to `Runner#paths_to_blueprints` - e.g.  `dredd.paths_to_blueprints('doc/*.apib')`
**When** I `run` Dredd 
**Then** I expect the local server URL to be defined automatically 
**And** the API to be tested 
**But** I get an `InvalidCommandError` instead 

See #17, where @rylnd wrote:

> Currently one is unable to do the following:
> ```ruby
Dredd::Rack::RakeTask.new do |task|
  task.runner.configure do |dredd|
    dredd.paths_to_blueprints 'doc/*.apib'
  end
end
```
> because the task exits early due to the # of arguments being checked _before_ we attempt to run the server (which has the side effect of adding an argument).

**Comment by rylnd** [comment]
:heart_eyes:

**Comment by gonzalo-bulnes** [comment]
Hi @rylnd,

I think I committed a mistake in **v0.7.0**, and I'm pretty sure the issue should not affect **v0.6.0**. (Could you confirm that?)

I think there is a much simpler way to fix the issue that the one you PR'd, let's disscuss it in [your there](https://github.com/gonzalo-bulnes/dredd-rack/pull/17) : )

**Comment by gonzalo-bulnes** [comment]
Hi @rylnd,

I took time, but I released **v0.8.2** with the fix for the inaccurate counting of arguments when testing local API. I used the solution you proposed (I was mistaken about the supposedly simpler solution), except for keeping the predicate methods. Thank you!

CC: @gottfrois

**Comment by rylnd** [comment]
👍 thanks @gonzalo-bulnes, nice work!
